helper/resource/testing_new_import_state.go

```diff
@@ -231,16 +231,40 @@ func testStepNewImportState(ctx context.Context, t testing.T, helper *plugintest
 		if err := runPlanChecks(ctx, t, plan, step.ImportPlanChecks.PreApply); err != nil {
 			return err
 		}
-	} else {
-		err = runProviderCommand(ctx, t, func() error {
-			return importWd.Import(ctx, resourceName, importId)
-		}, importWd, providers)
-		if err != nil {
-			return err
+
+		{
```

Comment:
Was this extra scope block meant to stay?

Reply:
For the moment, yes, but not for long. Like the TODO comment that comes after this logic branch, there's a bit of tidying we can do to extract chunks of work from the long function.

Reply:
Fair enough, just checking 😆